src/components/Datatable/Pagination/NavButton.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+import { getColor, pxToRem } from '../../../utils/helpers';
+import { IconButton } from '../../IconButton';
+import { IconButtonProps } from '../../IconButton/IconButton.types';
+
+const NavButton = styled(IconButton)
+  .attrs((props) => ({
+    ...props,
+  }))
+  .withConfig<IconButtonProps>({
+    shouldForwardProp: (prop) => prop !== 'disabled',
```

Comment:
Why are you disallowing forwarding of `disabled` prop? It's `<button>` native attribute which gives you the ability to actually disable button. So you should remove the `withConfig` and rather do:
```js
.attrs((props) => ({
    ...props,
    disabled: isDisabled
  }))